FBKVOControllerTests/FBKVOControllerTests.m

```diff
@@ -308,6 +308,26 @@ - (void)testObserveKeyPathsOptionsContextObservesEachOfTheKeyPaths
                                                   context:NULL];
 }
 
+- (void)testObserveKeyPathsOptionsContextUnobserveWithinBlockWithOptionInitial
```

Comment:
I am so sorry that I've missed this. This test is 4 spaces indented, where the code standard is 2 spaces :)
It's probably as simple as running auto-indent in Xcode...